docs/ai/conceptual/vector-dbs.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Using Vector Databases to Extend LLM Capabilities"
+description: "Learn how vector databases extend LLM capabilities by storing and processing embeddings in .NET."
+author: catbutler
+ms.topic: concept-article #Don't change.
+ms.date: 04/12/2024
+
+#customer intent: As a .NET developer, I want to learn how vector databases store and process embeddings in .NET so I can make more data available to LLMs in my apps.
+
+---
+
+# Understanding vector databases in .NET
+
+This article explains how vector databases help you use [embeddings](embeddings.md) to extend the data available to LLMs in .NET.
+
+You can use a vector database to store embeddings that you generate with AI embedding models. Embeddings have dimensions that correspond to the learned features or attributes of the embedding model you use. An embedding contains a value for each dimension, providing a semantic and mathematical representation of the source text.
+
+Vector databases can store embeddings for text, images, and other data types. You can then perform vector analysis on the embeddings to find semantic similarities in the source data, unlocking numerous AI use-cases.
+
+## Available vector database solutions
+
+You can use the following resources as vector database solutions in .NET:
+
+| Resource | SK support | AOAI support |
+|:-|:-|:-|
+| [Azure AI Search](/azure/search/vector-search-overview) | y | y |
+| [Azure Cache for Redis](/azure/azure-cache-for-redis/cache-tutorial-vector-similarity) | n | y |
+| [Azure Cosmos DB for MongoDB vCore](/azure/cosmos-db/mongodb/vcore/vector-search) | y | y |
+| [Azure Cosmos DB for NoSQL](/azure/cosmos-db/vector-search) | n | y |
+| [Azure Cosmos DB for PostgreSQL](/azure/cosmos-db/postgresql/howto-use-pgvector) | n | y |
+| [Azure Database for PostgreSQL - Flexible Server](/azure/postgresql/flexible-server/how-to-use-pgvector) | y | y |
+| [Azure SQL Database](/azure/azure-sql/database/ai-artificial-intelligence-intelligent-applications?&preserve-view=true#vector-search) | y | y |
+| [Open-source vector databases](/azure/cosmos-db/mongodb/vcore/vector-search-ai) | y | n |
+| | | |
```

Comment:
@gewarren are there standard icons to use for tables like this?

Reply:
Yes, use ✔️ and ❌ instead. Thanks for checking.

Reply:
Thanks! I couldn't remember how to make those appear.